docs/user-guide/styling-your-docs.md

```diff
@@ -30,12 +30,11 @@ the [custom themes](#custom-themes) section below.
 
 ![ReadTheDocs](https://docs.readthedocs.org/en/latest/_images/screen_mobile.png)
 
-## Bootstrap and Bootswatch Themes
+## Bootstrap and Bootswatch themes
```

Comment:
I have to admit, I didn't look at the rest of the document before making my comment. If you are following the style of the rest of the document, let's leave your change in for now. If we want to change the style, we can do that separately. However, that is @d0ugal 's call.